optic.yml

```diff
@@ -0,0 +1,20 @@
+ruleset:
```

Comment:
Are there alternatives to having this file present in the root? I'm thinking other locations or if it can be generated for plugins having openAPI specs for example and loaded from a temporary directory if it does not exist 🤷

Reply:
I'll chat with the Optic team about this, my thinking with this file is it lets us incrementally onboard plugins whereas running it against plugins that aren't using `createSupertestAgent` will cause errors. 

In that same vein, I think a temporary file or creating this as boilerplate would be very useful in letting the tests run in parallel by providing an open port and whatnot. However, it ends up in a similar problem where not all plugins with a spec are onboarded to use the new test agent (and that would be moreso in the wild --outside of this project--).

Reply:
I updated the `schema openapi test` command to create the file in the directory locally. I think it's ok to expect people that are using the tooling to set up a minimal amount of extra effort for added schema security. Curious to hear your thoughts? 

From what I can tell, Optic doesn't support config in a temporary directory and the options we need can't be passed in from CLI.

Reply:
I like the idea of having the file local to the plugin 👍